backend/src/test/java/capstoneexpensetracker/backend/TravelerIntegrationTest.java

```diff
@@ -0,0 +1,32 @@
+package capstoneexpensetracker.backend;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class TravelerIntegrationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+    @Autowired
```

Comment:
Lieber Autowired in Integrationtests nur für MockMvc nutzen. Felder wie ObjectMapper einfach im Test instanziieren, damit der Test unabhängig vom Production Code bleibt.